### PR TITLE
#11017 #4 Use EmbeddedBroadcast service in base player controller

### PR DESCRIPTION
**Pull request by mreyk**
Instead of testBroadcast

### CODE REVIEW

**Comment by mreyk** [comment]
Fixes #4

File: Controller/BasePlayerController.php [hunk 1]
```diff
@@ -52,7 +54,9 @@ public function magicAction(MultimediaObject $multimediaObject, Request $request
      */
     public function indexAction(MultimediaObject $multimediaObject, Request $request)
     {
-        $response = $this->testBroadcast($multimediaObject, $request);
+        $embeddedBroadcastService = $this->get('pumukitschema.embeddedbroadcast');
+        $password = $request->get('broadcast_password');
+        $response = $embeddedBroadcastService->canUserPlayMultimediaObject($multimediaObject, $this->getUser(), $password, $request->query->get('force-auth'));
```
Comment (rubenrua) [inline Controller/BasePlayerController.php]: `force-auth` is not needed.

See https://github.com/campusdomar/PuMuKIT2/commit/8d074b045ee28f3e609e645e027b93a949180291

File: Controller/BasePlayerController.php [hunk 2]
```diff
@@ -21,7 +21,9 @@ public function magicAction(MultimediaObject $multimediaObject, Request $request
             return $this->redirect($this->generateUrl('pumukit_videoplayer_magicindex', array('id' => $multimediaObject->getSecret(), 'secret' => $multimediaObject->getSecret())).'&secret='.$multimediaObject->getSecret());
         }
 
-        $response = $this->testBroadcast($multimediaObject, $request);
+        $embeddedBroadcastService = $this->get('pumukitschema.embeddedbroadcast');
+        $password = $request->get('broadcast_password');
+        $response = $embeddedBroadcastService->canUserPlayMultimediaObject($multimediaObject, $this->getUser(), $password, $request->query->get('force-auth'));
```
Comment (rubenrua) [inline Controller/BasePlayerController.php]: `force-auth` is not needed.

See https://github.com/campusdomar/PuMuKIT2/commit/8d074b045ee28f3e609e645e027b93a949180291

**Comment by mreyk** [comment]
Close this and do this pull request instead:
https://github.com/teltek/PuMuKIT2-paella-player-bundle/pull/14

**Pull request closed without merging**